test_runner/logical_repl/test_log_repl.py

```diff
@@ -86,3 +136,108 @@ def test_clickhouse(remote_pg: RemotePostgres):
     log.debug("Sleeping before final checking if Neon is still alive")
     time.sleep(3)
     cur.execute("SELECT 1")
+
+
+@pytest.fixture(scope="function")
+def debezium(remote_pg):
+    """
+    Prepare the Debezium API handler, connection
+    """
+    conn = psycopg2.connect(remote_pg.connstr())
+    cur = conn.cursor()
+    cur.execute("DROP SCHEMA IF EXISTS inventory CASCADE")
+    cur.execute("CREATE SCHEMA inventory")
+    cur.execute(
+        "CREATE TABLE inventory.customers ("
+        "id SERIAL NOT NULL PRIMARY KEY,"
+        "first_name character varying(255) NOT NULL,"
+        "last_name character varying(255) NOT NULL,"
+        "email character varying(255) NOT NULL)"
+    )
+    conn.commit()
+    dbz = DebeziumAPI("http://debezium:8083")
+    assert len(dbz.list_connectors()) == 0
+    dbz_conn_name = "inventory-connector"
+    resp = dbz.create_pg_connector(remote_pg, dbz_conn_name)
+    log.debug("%s %s %s", resp.status_code, resp.ok, resp.text)
+    assert resp.status_code == 201
+    assert len(dbz.list_connectors()) == 1
+    consumer = KafkaConsumer(
+        "dbserver1.inventory.customers",
+        bootstrap_servers=["kafka:9092"],
+        auto_offset_reset="earliest",
+        enable_auto_commit=False,
+    )
+    yield conn, consumer
+    resp = dbz.del_connector(dbz_conn_name)
+    assert resp.status_code == 204
+
+
+def get_kafka_msg(consumer, ts_ms, before=None, after=None) -> None:
+    """
+    Gets the message from Kafka and checks its validity
+    """
+    msg = consumer.poll()
+    if not msg:
+        raise ValueError("Empty message")
+    for val in msg.values():
+        r = json.loads(val[-1].value)
+        log.info(r["payload"])
+        if ts_ms >= r["payload"]["ts_ms"]:
+            log.info("incorrect ts")
+            raise ValueError("Incorrect timestamp")
```

Comment:
Should we use an assert here?
```suggestion
        assert ts_ms < r["payload"]["ts_ms"], "Incorrect timestamp"
```

Reply:
wait_until does not print the cause of failure.

Reply:
done